build.gradle

```diff
@@ -22,18 +22,18 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
     implementation 'org.springframework.boot:spring-boot-starter-reactor-netty'
-    implementation 'org.springframework:spring-webflux:5.3.15'
-    implementation 'org.springframework:spring-webflux:5.3.15'
-    implementation 'uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.0.2'
+    implementation 'org.springframework:spring-webflux:5.3.16'
+    implementation 'org.springframework:spring-webflux:5.3.16'
```

Comment:
do you need to specify this version? most of the other repos don't, would the correct version be inherited anyway?